docs/rackhd/passive_discovery.rst

```diff
@@ -65,6 +66,45 @@ in the graph options, eg:
 
     curl -X POST \
         -H 'Content-Type: application/json' \
-        -d '{"name": "Graph.Switch.Discovery",
+        -d '{"name": "Graph.Switch.Discovery", \
+         "options":{"defaults":{"nodeId": "55b6afba024fd1b349afc148"}}}' \
+        <server>/api/1.1/nodes/55b6afba024fd1b349afc148/workflows
+
+You can also use this mechanism to discovery a compute server or PDU, simply
+using different settings. For example, a smart PDU:
+
+.. code-block:: REST
+
+    curl -X POST \
+        -H 'Content-Type: application/json' \
+        -d '{"name":"nodeName", "type": "pdu", \
+         "snmpSettings": {"host": "10.1.1.3", "community": "public"}}' \
+        <server>/api/1.1/nodes
+
+.. code-block:: REST
+
+    curl -X POST \
+        -H 'Content-Type: application/json' \
+        -d '{"name": "Graph.PDU.Discovery", \
+         "options":{"defaults":{"nodeId": "55b6afba024fd1b349afc148"}}}' \
+        <server>/api/1.1/nodes/55b6afba024fd1b349afc148/workflows
+
+And a management server (or other server you do not want to or ca not to reboot
+to interrogate)
+
+.. code-block:: REST
+
+    curl -X POST \
+        -H 'Content-Type: application/json' \
+        -d '{"name":"nodeName", "type": "compute", \
+         "snmpSettings": {"host": "10.1.1.3", "user": "admin",  \
+         "pass": "admin"}}' \
```

Comment:
`ipmi-obm-setting` I think was specifically "pass" and not "password" - the later resulted in a failure to operate correctly. I'll update the snmpSetting though